[DOCS-1751] Reorganize and correct docs about viewing and exporting user activity (#1627)
wandb/docs: content/en/guides/hosting/monitoring-usage/org_dashboard.md

title: View organization activity
---

> This page describes how to monitor activity in your W&B organization. Select your deployment type to continue.

Reviewer: minuscule nit (feel free to ignore): "view" feels like a more accurate word than monitor. To me, a monitor is used by automations...

> This page describes how to monitor activity in your W&B organization. Select your deployment type to continue.

Author: ```suggestion
This page describes how to view activity in your W&B organization. Select your deployment type to continue.
```

## View user status and activity 

{{< tabpane text=true >}}

> {{% tab header="Self-Managed" value="dedicated" %}}

Author: ```suggestion
{{% tab header="Dedicated Cloud / Self-Managed" value="dedicated" %}}
```

1. Navigate to the **Organization Dashboard** at `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default and lists every user in the organization.
2. To sort the list by user status, click the **Last Active** column header. Each user's status is one of the following:
   * **Invite pending** – an invitation was sent but not yet accepted.

>    * **Active** – the user accepted the invitation and has signed in at least once.

Reviewer: nit: Active isn't a status. The Timestamp of the user's last activity is shown instead.

>    * **Active** – the user accepted the invitation and has signed in at least once.

Author: ```suggestion
   * **A timestamp** – the user accepted the invitation and has signed in at least once. The timestamp indicates the most recent activity.
```

   * **–** – the user was previously active but has not been active in the last six months.
   * **Deactivated** – an admin revoked the user's access.
3. Hover over a user's **Last Active** field to see the date the user was added and their total active days.

A user is _active_ if they:
- sign in to W&B.
- open any page in the W&B App.
- log runs.
- use the SDK to track an experiment.
- interact with the W&B server in any way.
{{% /tab %}}

{{% tab header="Multi-tenant Cloud" value="saas" %}}

> 1. Open the [**Members** page](https://wandb.ai/account-settings/wandb/members/). The table lists every user in your organization.

Reviewer: can you replace the URL with https://wandb.ai/account-settings/wandb/users and call it the Users tab so the wording matches the UI?

Author: We discussed, and decided no.

2. Click the **Last Active** column header to sort by user status. Each user's status is one of the following:

>    * **Invite pending** – an invitation was sent but not yet accepted.

Reviewer: There's only two options for the contents of the last active column

1. - (the user has not yet been active within the org)
2. the date the user was last active

>    * **Invite pending** – an invitation was sent but not yet accepted.

Author: ```suggestion
```

   * **Active** – the user accepted the invitation and has signed in at least once.
   * **–** – the user has not yet been active within the organization.

A user is _active_ if they perform any auditable action scoped to the organization _after May 8 2025_. For a full list, see [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in **Audit logging**.
{{% /tab %}}
{{< /tabpane >}}

## View activity over time  {#view-activity-over-time-dedicated}
{{< tabpane text=true >}}

> {{% tab header="Self-Managed" value="dedicated" %}}

Author: ```suggestion
{{% tab header="Dedicated Cloud / Self-Managed" value="dedicated" %}}
```

2. Click **Activity**.
3. Review the following plots:
   * **Total active users** – unique active users during the selected period (defaults to 3 months).
   * **Users active over time** – fluctuation of active users over the period (defaults to 6 months). Hover over a point to see the exact count on that date.

To change the period, use the drop-down above a plot. Options are Last 30 days, Last 3 months, Last 6 months, Last 12 months, and All time.
{{% /tab %}}
{{% tab header="Multi-tenant Cloud" value="saas" %}}
Use the **Activity Dashboard** to view aggregate activity.

1. Click your user icon in the upper-right corner of the W&B App.
2. Under **Account**, click **Users**.
3. Above the table of users, review the Activity Panel:
   * **Active user count** – unique active users during the selected period (defaults to 3 months).
   * **Weekly active users** – users active per week.
   * **Most active user** – top-10 users ranked by active days and last-active date.
4. To change the date range (7, 30, or 90 days; default is 30 days), click the date picker in the upper-right corner. All plots update automatically.
{{% /tab %}}
{{< /tabpane >}}

> ## Export user details

Author: ```suggestion
## Export user details

From the **Users** tab you can download a CSV that lists each user's details (user name, email address, last-active time, roles, and more).

```

{{< tabpane text=true >}}

> {{% tab header="Self-Managed" value="dedicated" %}}

Author: ```suggestion
{{% tab header="Dedicated Cloud / Self-Managed" value="dedicated" %}}
```

> From the **Users** tab you can download a CSV that lists each user's details (user name, email address, last-active time, roles, and more).

Author: ```suggestion
```

1. In the **Users** tab, click the **…** actions menu next to **Invite new user**.
2. Click **Export as CSV**.
{{% /tab %}}

{{% tab header="Multi-tenant Cloud" value="saas" %}}

> You can export the table of users to CSV.

Reviewer: nit: why not repeat the contents for single-tenant?
From the **Users** tab you can download a CSV that lists each user's details (user name, email address, last-active time, roles, and more).

> You can export the table of users to CSV.

Author: ```suggestion
```
Done, moved it above the tabs.

> 1. On the **Members** page, click the **…** actions menu in the upper-right corner.

Reviewer: the ui and url call this page "Users" page
https://wandb.ai/account-settings/wandb/users

Author: It's the **Users** tab but it is the **Members** page, says the URL.

> 1. On the **Members** page, click the **…** actions menu in the upper-right corner.

Author: ```suggestion
1. In the **Users** tab, click the **…** actions menu in the upper-right corner.
```

2. Select **Export as CSV** to download the file.
{{% /tab %}}

> {{< /tabpane >}}

Author: ```suggestion
{{< /tabpane >}}

The CSV export uses the comma (`,`) as the separator, encloses strings in double quotes, and includes a header row that defines these columns:

- `"Name"`
- `"Username"`
- `"Last active"`
- `"Role"`
- `"Email"`
- `"Teams"`
- `"Status"`
- `"Number of Reports"`
- `"Number of Runs"`
- `"Number of active days"`
- `"Models Seat"`
```

title: View organization activity
---

This page describes how to view activity in your W&B organization. Select your deployment type to continue.

## View user status and activity 

{{< tabpane text=true >}}
{{% tab header="Dedicated Cloud / Self-Managed" value="dedicated" %}}
1. Navigate to the **Organization Dashboard** at `https://<org-name>.io/org/dashboard/`. Replace `<org-name>` with your organization name. The **Users** tab opens by default and lists every user in the organization.
2. To sort the list by user status, click the **Last Active** column header. Each user's status is one of the following:

>    * **Invite pending** – an invitation was sent but not yet accepted.

Reviewer 2: Do you mean to use colons instead of dashes (applies here and the other tab below, too)?

   * **A timestamp** – the user accepted the invitation and has signed in at least once. The timestamp indicates the most recent activity.

>    * **–** – the user was previously active but has not been active in the last six months.

Reviewer 2: Bolding a single dash here renders weird in the UI (see doc preview). Can you use words and then add in paranthesis what it'll look like in the UI?

   * **Deactivated** – an admin revoked the user's access.
3. Hover over a user's **Last Active** field to see the date the user was added and their total active days.

A user is _active_ if they:
- sign in to W&B.
- open any page in the W&B App.
- log runs.
- use the SDK to track an experiment.
- interact with the W&B server in any way.
{{% /tab %}}

{{% tab header="Multi-tenant Cloud" value="saas" %}}
1. Open the [**Members** page](https://wandb.ai/account-settings/wandb/members/). The table lists every user in your organization.
2. Click the **Last Active** column header to sort by user status. Each user's status is one of the following:

>    * **Active** – the user accepted the invitation and has signed in at least once.

Reviewer: this should say: 

* **A timestamp** – the user has signed in at least once. The timestamp indicates the most recent activity.

   * **–** – the user has not yet been active within the organization.

> A user is _active_ if they perform any auditable action scoped to the organization _after May 8 2025_. For a full list, see [Actions]({{< relref "/guides/hosting/monitoring-usage/audit-logging.md#actions" >}}) in **Audit logging**.

Reviewer 2: Suggestion: remove italicization for "after May 8 2025". It diminshes the impact of the other terms here that are italicized.

1. Click your user icon in the upper-right corner of the W&B App.
2. Under **Account**, click **Users**.
3. Above the table of users, review the Activity Panel:
   * **Active user count** – unique active users during the selected period (defaults to 3 months).
   * **Weekly active users** – users active per week.
   * **Most active user** – top-10 users ranked by active days and last-active date.
4. To change the date range (7, 30, or 90 days; default is 30 days), click the date picker in the upper-right corner. All plots update automatically.
{{% /tab %}}
{{< /tabpane >}}

## Export user details

From the **Users** tab you can download a CSV that lists each user's details (user name, email address, last-active time, roles, and more).

{{< tabpane text=true >}}
{{% tab header="Dedicated Cloud / Self-Managed" value="dedicated" %}}

1. In the **Users** tab, click the **…** actions menu next to **Invite new user**.
2. Click **Export as CSV**.
{{% /tab %}}

{{% tab header="Multi-tenant Cloud" value="saas" %}}

1. In the **Users** tab, click the **…** actions menu in the upper-right corner.
2. Select **Export as CSV** to download the file.
{{% /tab %}}
{{< /tabpane >}}

> The CSV export uses the comma (`,`) as the separator, encloses strings in double quotes, and includes a header row that defines these columns:

Reviewer: Forgive me if I'm reading the markdown wrong, but line 78 suggests this is a description for SaaS.

The columns described here are only accurate for Single-tenant.

(SaaS has different values.)

Author: @niall-wb told me in Slack that both Dedicated and SaaS use the same export schema so let's double-check it to make sure I was not confused.

Author: Still trying to resolve this, it would be good to get this PR in ahead of the content freeze next week.

Reviewer 3: It looks like they ARE different. Here's what I found in the code: Yes, they ARE different between versions.

SaaS 
  - "Name"
  - "Username"
  - "Org Role"
  - "Models Seat"
  - "Weave Access"
  - "Email"
  - "Teams"
  - "Last active"

Single-tenant
  - "Name"
  - "Username"
  - "Last active"
  - "Role"
  - "Email"
  - "Teams"
  - "Status"
  - "Number of Reports"
  - "Number of Runs"
  - "Number of active days"
  - "Models Seat"
  - "Weave Access" (conditional)
